m/mullvad/stone.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# SPDX-FileCopyrightText: © 2020-2025 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : mullvad
+version     : 2025.8
+release     : 1
+homepage    : https://mullvad.net
+upstreams   :
+    - git|https://github.com/mullvad/mullvadvpn-app.git : 94fee68b5a83fe1df0cbc99a3d303fd7ef15519f
+summary     : The Mullvad VPN client app for desktop
+description : |
+    The Mullvad VPN client app for desktop
+license     : GPL-3.0-or-later
+networking  : true
+builddeps   :
+    - pkgconfig(alsa)
+    - pkgconfig(gbm)
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(protobuf)
+    - gcc
+    - git
+    - golang
+    - nodejs
+    - rust
```

Comment:
It looks like you just copied the Solus builddeps and added gcc. Please don't do this, you can use the solus recipe as a starting point but you should always adjust it to use aeryn packages and providers. For instance, the following works from my testing:
```
builddeps   :
    - binary(ar)
    - binary(go)
    - binary(npm)
    - binary(protoc)
    - binary(rustc)
    - pkgconfig(dbus-1)
    - soname(libgcc_s.so.1(x86_64))
```